main/src/ca/uwaterloo/flix/language/ast/SourceLocation.scala

```diff
@@ -82,7 +82,10 @@ case class SourceLocation(input: Option[ParserInput], source: Source, locationKi
     */
   def lineAt(line: Int): String = input match {
     case None => ""
-    case Some(input) => input.getLine(line)
+    case Some(input) =>
+      input.getLine(line)
+        .replaceAll(System.lineSeparator(), "")
```

Comment:
This has to be \n. You cannot assume that platform has \n as separator, so we have to write \n and \r

Reply:
:)

Reply:
Roger roger. Is \n used the source location instead of the platform independent line separator?